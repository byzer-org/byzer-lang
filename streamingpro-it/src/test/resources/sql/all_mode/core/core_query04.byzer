load parquet.`/ssb/customer/part-00000-7505a64d-b4bf-4186-874a-a9898502a51e-c000.snappy.parquet` as customer_parquet;

select * from customer_parquet as customer;

select C_ADDRESS from customer limit 5 as customer_default;

select upper(C_ADDRESS) as C_ADDRESS_UPPER from customer_default as customer_upper;
