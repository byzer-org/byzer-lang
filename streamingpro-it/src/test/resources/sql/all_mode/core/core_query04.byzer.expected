C_ADDRESS_UPPER
J5JSIRBM9P
487LW1DOVN6Q4DMVYM
FKRGN8N
4U58H F
HWBTXKOBF QSW4KRI